OCPP-J/src/main/java/eu/chargetime/ocpp/JSONConfiguration.java

```diff
@@ -49,8 +49,10 @@ public class JSONConfiguration {
 
   private JSONConfiguration() {}
 
+  private static final JSONConfiguration instance = new JSONConfiguration();
+
   public static JSONConfiguration get() {
-    return new JSONConfiguration();
+    return instance;
```

Comment:
API breaking change. Instead of getting a default configuration, you now get whatever configuration the last library user left behind.